services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -284,6 +287,88 @@ async def retrieve_service_inputs(
 
         return RetrieveDataOutEnveloped.from_transferred_bytes(transferred_bytes)
 
+    async def attach_sharing_network(
+        self, node_id: NodeID, network_id: str, network_alias: DockerNetworkAlias
+    ) -> None:
+        if node_id not in self._inverse_search_mapping:
+            logger.debug("No dynamic-sidecar running for %s", f"{node_id=}")
+            return
+
+        service_name = self._inverse_search_mapping[node_id]
+        scheduler_data = self._to_observe[service_name].scheduler_data
+
+        dynamic_sidecar_client: DynamicSidecarClient = get_dynamic_sidecar_client(
+            self.app
+        )
+
+        try:
+            containers_status = await dynamic_sidecar_client.containers_docker_status(
+                dynamic_sidecar_endpoint=scheduler_data.dynamic_sidecar.endpoint
+            )
+        except httpx.HTTPError:
+            logger.debug("Could not contact dynamic-sidecar for %s", f"{node_id=}")
```

Comment:
just a debug message?
what will happen then? does it retry later?

Reply:
Yes, I can put it to info.
The api can get called and the dynamic-sidecar may not be running. It is ok to do nothing in this case.

Reply:
well then if this is useless, you should just suppress the message. it's cleaner.